config/routes/system_admin_routes.rb

```diff
@@ -68,6 +68,14 @@ def self.extended(router)
           resources :retrieve_trns, only: %i[update], path: "retrieve-trns"
           resources :request_trns, only: %i[update], path: "request-trns"
         end
+
+        resources :trainees, only: [], path: "trainees" do
```

Comment:
I wondered whether this file was the correct location for these routes. They're operated via system admins but they sit within the non support flow so feels like they should go in the other file, then we can just rely on finding the trainee via the slug to be consistent with everywhere else. Otherwise we'll have to change and explicitly pass in a trainee id

Reply:
I wasn't sure. I ended up putting it under the `system-admin` routes because it would then automatically get the same authorisation check that other such routes get to stop non-admins using this feature. But using the `slug_id` sounds like a better way to go TBH...

Reply:
yeah a bit unusual. I leave it with you to make the call. We can always card it up as a follow on ticket to refactor? At least this way this ticket will unblock the others